front-end/src/components/App.js

```diff
@@ -55,13 +58,19 @@ const App = () => {
     type: "line"
   });
 
+  const [successMessage, setSuccessMessage] = useState("")
+
+  const success = () => {
+    $( "div.success" ).fadeIn(1000).delay(2000).fadeOut(1000)
+  };
```

Comment:
Didn't find anything that looked cooler than fade in?

Reply:
I actually dont think there is another option with jquery. At least none that I found. Personally I like it so I dont see the need to change anything